products/ipad.md

```diff
@@ -0,0 +1,152 @@
+    releaseLabel: "iPad Air (5th generation)"
+    releaseDate: 2022-03-18
+    discontinued: false
+    eol: false
+
+-   releaseCycle: "9"
+    releaseLabel: "iPad (9th generation)"
+    releaseDate: 2021-09-24
+    discontinued: false
+    eol: false
+
+-   releaseCycle: "mini-6"
+    releaseLabel: "iPad Mini (6th generation)"
+    releaseDate: 2021-09-24
+    discontinued: false
+    eol: false
+
+-   releaseCycle: "8"
+    releaseLabel: "iPad (8th generation)"
+    releaseDate: 2020-09-18
+    discontinued: 2021-09-14
+    eol: false
+
+-   releaseCycle: "air-4"
+    releaseLabel: "iPad Air (4th generation)"
+    releaseDate: 2020-10-23
+    discontinued: 2022-03-08
+    eol: false
+
+-   releaseCycle: "7"
+    releaseLabel: "iPad (7th generation)"
+    releaseDate: 2019-09-25
+    discontinued: 2020-09-15
+    eol: false
+
+-   releaseCycle: "mini-5"
+    releaseLabel: "iPad Mini (5th generation)"
+    releaseDate: 2019-03-18
+    discontinued: 2021-09-14
+    eol: false
+
+-   releaseCycle: "air-3"
+    releaseLabel: "iPad Air (3rd generation)"
+    releaseDate: 2019-03-18
+    discontinued: 2021-09-15
+    eol: false
+
+-   releaseCycle: "6"
+    releaseLabel: "iPad (6th generation)"
+    releaseDate: 2018-03-27
+    discontinued: 2019-09-10
+    eol: false
+
+-   releaseCycle: "5"
+    releaseLabel: "iPad (5th generation)"
+    releaseDate: 2017-03-24
+    discontinued: 2018-03-27
+    eol: false
+
+-   releaseCycle: "mini-4"
+    releaseLabel: "iPad Mini 4"
+    releaseDate: 2015-09-09
+    discontinued: 2019-03-18
+    eol: false
+
+-   releaseCycle: "mini-3"
+    releaseLabel: "iPad Mini 3"
+    releaseDate: 2014-10-22
+    discontinued: 2015-09-09
+    eol: 2022-08-31
+
+-   releaseCycle: "air-2"
+    releaseLabel: "iPad Air 2"
+    releaseDate: 2014-10-22
+    discontinued: 2017-03-21
+    eol: false
+
+-   releaseCycle: "mini-2"
+    releaseLabel: "iPad Mini 2"
+    releaseDate: 2013-11-12
+    discontinued: 2017-03-21
+    eol: 2022-08-31
+
+-   releaseCycle: "air-1"
+    releaseLabel: "iPad Air (1st generation)"
+    releaseDate: 2013-11-01
+    discontinued: 2016-03-21
+    eol: 2022-08-31
+
+-   releaseCycle: "4"
+    releaseLabel: "iPad (4th generation)"
+    releaseDate: 2012-11-02
+    discontinued: 2014-10-16
+    eol: 2019-07-22
+
+-   releaseCycle: "3"
+    releaseLabel: "iPad (3rd generation)"
+    releaseDate: 2012-03-16
+    discontinued: 2012-10-23
+    eol: 2019-07-22
+
+-   releaseCycle: "mini-1"
+    releaseLabel: "iPad Mini (1st generation)"
+    releaseDate: 2012-11-02
+    discontinued: 2015-06-19
+    eol: 2019-07-22
+
+-   releaseCycle: "2"
+    releaseLabel: "iPad 2"
+    releaseDate: 2011-03-11
+    discontinued: 2014-03-18
+    eol: 2019-07-22
+
+-   releaseCycle: "1"
+    releaseLabel: "iPad (1st generation)"
+    releaseDate: 2010-04-03
+    discontinued: 2011-03-02
+    eol: 2012-09-19
+
+---
+
+> The iPad is a line of tablet based computers designed and marketed by Apple Inc. that use Apple's
```

Comment:
@captn3m0, I think this description should mention clearly that iPad is available in 3 "formats" : iPad "classic", iPad Air and iPad Mini.

Reply:
Could add these, but I'm not sure if it's gonna be helpful. 

It should be more obvious that the page includes all form factors, but that's covered reasonably by the table. Not sure if the description needs to change for that.